src/Compilers/CSharp/Test/Symbol/DocumentationComments/MethodDocumentationCommentTests.cs

```diff
@@ -231,12 +231,12 @@ class Test
             var compilation = CreateCompilation(source, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular.WithDocumentationMode(DocumentationMode.Diagnose));
             var main = compilation.GetTypeByMetadataName("Test").GetMember<MethodSymbol>("Main");
 
-            Assert.Equal(@"<!-- Badly formed XML comment ignored for member ""M:Test.Main"" -->", main.GetDocumentationCommentXml().Trim());
+            Assert.Equal(string.Format(CSharpResources.IDS_XMLIGNORED, "M:Test.Main"), main.GetDocumentationCommentXml().Trim());
```

Comment:
>Assert.Equal(string.Format(CSharpResources.IDS_XMLIGNORED, "M:Test.Main"), main.GetDocumentationCommentXml().Trim()); [](start = 12, length = 117)

Please change UI culture for the duration of this function instead. using (new EnsureEnglishUICulture())